### PR TITLE
chore(northlight): add icon placement prop

### PR DESCRIPTION
**Pull request by unganemo**
Added an icon placement prop so user can decide if icon should be placed on the left or right side of the button.
if iconPlacement is none, no icon will be shown (including checkmark).
iconPlacement defaults to left.

closes: DEV-8553

### CODE REVIEW

File: framework/lib/components/flip-button/flip-button-group.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ export const FlipButtonGroup = (props: FlipButtonGroupProps) => {
     children,
     direction = 'row',
     isDisabled = false,
+    hasCheckmark = true,
```
Comment (TautvydasGec) [inline framework/lib/components/flip-button/flip-button-group.tsx:20]: Also need it on FlipBUttonGroupField.

**Review by TautvydasGec — COMMENTED** [review]
@ilija- what about we would add a prop like selectedIcon. we could pass 'left' | 'rigth' | 'none' ?

File: framework/lib/components/flip-button/flip-button-group.tsx (hunk 1 above)
Comment (ilija-) [reply framework/lib/components/flip-button/flip-button-group.tsx:20]: This is actually not a bad idea

**Review by ilija- — APPROVED** [review]

**Comment by TautvydasGec** [comment]
@ilija- could you check my comment?

File: framework/lib/components/flip-button/flip-button-group.tsx (hunk 1 above)
Comment (ilija-) [reply framework/lib/components/flip-button/flip-button-group.tsx:20]: Sorry i missed that one. 

`showIcon: 'left' | 'rigth' | 'none' ` seems pretty understandable to me.

Comment (unganemo) [reply framework/lib/components/flip-button/flip-button-group.tsx]: @ilija- check latest
`iconPos: 'left' | 'right' | 'none'`

Comment (ilija-) [reply framework/lib/components/flip-button/flip-button-group.tsx]: Position none ?

Comment (TautvydasGec) [reply framework/lib/components/flip-button/flip-button-group.tsx]: showIcon implies action so it should be a function. @ilija-

Comment (ilija-) [reply framework/lib/components/flip-button/flip-button-group.tsx]: hmm .. then I'm okay with that. `iconPos` .. seems plausible.

**Review by ilija- — APPROVED** [review]

**Review by TautvydasGec — APPROVED** [review]

**Comment by ilija-** [comment]
> @ilija- what about we would add a prop like selectedIcon. we could pass 'left' | 'rigth' | 'none' ?

Too long too much. The one above is fine IMO, but if you guys decide to go with that one I'm fine. Either way, it will serve the developers.

**Pull request merged**